### PR TITLE
Client validation

### PR DESCRIPTION
**Pull request by lewisjn-arch**
This PR has:

- [ ] Client side validation in javascript and built-in validation,
- [ ] Error message styling.
- [ ] Updated index.html.

### CODE REVIEW

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @lewisjn-arch, and @Cheptoo21 ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be Invalidated and won't affect the number of your weekly reviews.

- [ ] Kindly do not disable linter checks as they help us write more professional code. Kindly remove the linter disabling code from your `validation.js` file.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: validation.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+function validate() {
+  if (!validateName() || !validateEmail() || !validateText()) {
+    submitError.style.display = 'block';
+    submitError.innerHTML = 'Please fix errors in the required fields!';
+    setTimeout(() => { submitError.style.display = 'none'; }, 3000);
+    return false;
+  }
+  return true;
+}
```
Comment (Cabraham1) [inline validation.js]: - [ ] Great job so far! 👍🏿 I appreciate the effort you put into the validation. However, I'd like to remind you that in a previous milestone, you set a validation for the text field with a `max-length` of 500 characters. Therefore, adding another validation for the same field goes against the requirements for this current milestone. As per the project requirements, we only need email validation. So, I kindly suggest removing the text validation and focusing solely on email validation. Keep up the good work! 🥇

**Review by Cabraham1 — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @lewisjn-arch  :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:
- [ ] I kindly request that you adhere to the Figma design provided for your project.  as you can see from the screenshot your input text has too much padding i kindly suggest you align the text to the right 👍🏿 
image


# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers, and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Form validation works correctly👍 


## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**